chromium_src/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc

```diff
@@ -105,19 +106,7 @@ DontAskAgainCheckbox::DontAskAgainCheckbox(WidevinePermissionRequest* request)
       request_(request) {}
 
 void DontAskAgainCheckbox::ButtonPressed() {
-  request_->set_dont_ask_widevine_install(GetChecked());
-}
-
-bool HasWidevinePermissionRequest(
-    const std::vector<permissions::PermissionRequest*>& requests) {
-  // When widevine permission is requested, |requests| only includes Widevine
-  // permission because it is not a candidate for grouping.
-  if (requests.size() == 1 &&
-      requests[0]->request_type() == permissions::RequestType::kWidevine) {
-    return true;
-  }
-
-  return false;
+  request_->SetDontAskAgain(GetChecked());
```

Comment:
I don't understand what is happening here, there seems to be a new method, but the old method still exists? Also the old method followed the correct naming convention for simple setter/getter methods and this one does not

Reply:
The old one is deleted in `browser/widevine/widevine_permission_request.h`, check line 31. The new one is not a one liner in header but a normal function. In that case it shouldn't be lower cased I think?

Reply:
https://github.com/brave/brave-core/pull/18958/files#diff-975e8695a60247d579b6e739fb41fa3e24be59662c6f5fdda25c45c5688fc2d6R126 is not a one-liner?

Reply:
I thought lower case function name only applicable if the whole function is in header? I'm not super familiar with C++ naming conventions so I may be wrong. In any case I can just make them lower case.

Reply:
Fixed.